README.md

```diff
@@ -84,17 +84,82 @@ The primary supported way of running PeopleMath in a staging or production confi
 
-To enable authorization with [Firebase](https://firebase.google.com/docs/auth) to authenticate users:
+A new team's permission will default to the general, app-wide permissions
+(the `Read` list will mirror the `ReadTeamList` list and the `Write` list will mirror the `AddTeam` list)
+
+To use authorization with [Firebase](https://firebase.google.com/docs/auth) to authenticate users:
 
 * Set `requireAuth` to `true` in `src/environments/environment.prod.ts`
 * Run backend with authmode flag `--authmode firebase`
 * Download your [Firebase config file](https://support.google.com/firebase/answer/7015592#web) and copy the text into the `firebaseConfig.ts` file in the same folder.
 * Download your [Firebase credentials](https://firebase.google.com/docs/admin/setup#initialize-sdk) and export them via `export GOOGLE_APPLICATION_CREDENTIALS="/path/to/your/credentials.json"`
-* Set the email domain for the authorization using the `authdomain` flag. The default is `google.com`
+* Add the domain which the AppEngine instance runs on the OAuth redirect domains list in the Firebase console (to get there: Firebase Console -> Authentication section -> Sign in method tab)
+* Set the general permissions in the Google Cloud Data Store after starting the AppEngine instance following this example
+(to get there: (Google Cloud Platform -> Datastore -> Entities -> select Kind: Settings -> settings entity details on the right -> edit -> edit GeneralPermissions property)):
+```json
+{
+  "ReadTeamList": {
+    "entityValue": {
+      "properties": {
+        "Allow": {
+          "arrayValue": {
+            "values": [
+              {
+                "entityValue": {
+                  "properties": {
+                    "Type": {
+                      "stringValue": "domain"
+                    },
+                    "ID": {
+                      "stringValue": "google.com"
+                    }
+                  }
+                }
+              }
+            ]
+          }
+        }
+      }
+    }
+  },
+  "AddTeam": {
+    "entityValue": {
+      "properties": {
+        "Allow": {
+          "arrayValue": {
+            "values": [
+              {
+                "entityValue": {
+                  "properties": {
+                    "Type": {
+                      "stringValue": "domain"
+                    },
+                    "ID": {
+                      "stringValue": "google.com"
+                    }
+                  }
+                }
+              }
+            ]
+          }
+        }
+      }
+    }
+  }
+}
+```
 
 Currently, the only authentication method enabled on Firebase is Google Sign-in.
 
+To use in memory storage with authentication and authorization,
+set the defaultdomain flag with your account's domain `--defaultdomain yourdomain.com`
+
+Because the permissions have to be set manually, there is currently no way to use authentication and authorization with the Google Cloud emulator.
```

Comment:
I'm not sure this is true, is it? Why can't you just connect to Firebase Auth from your development machine using service account credentials, and manually insert permissions into the (Cloud Datastore emulator based) database like you have suggested doing above?

OK, you won't be able to use the Pantheon UI, but you should be able to post these JSON entries into the API using `curl` or equivalent.

Reply:
Ah, okay, my bad! I was trying to find answers online whether you can update the database when using the emulator and couldn't find any answers but what you're saying is probably true. I now found some posts that indicate it is possible, but I don't understand it well enough to be able to update the README.